astropy/modeling/polynomial.py

```diff
@@ -1111,13 +1149,14 @@ def _chebderiv1d(self, x, deg):
 
 
 class Legendre2D(OrthoPolynomialBase):
-    """
```

Comment:
Whoops, sorry! (Haven't had coffee yet!) The failure still seems real on Appveyor though.